feature-libs/cart/base/components/added-to-cart-toast/added-to-cart-toast.component.ts

```diff
@@ -0,0 +1,202 @@
+import {
+  ChangeDetectorRef,
+  Component,
+  ElementRef,
+  HostBinding,
+  OnDestroy,
+  Renderer2,
+} from '@angular/core';
+import { WindowRef } from '@spartacus/core';
+import { Observable, OperatorFunction, Subscription } from 'rxjs';
+import { filter, take } from 'rxjs/operators';
+import {
+  ActiveCartFacade,
+  CartToastItem,
+  CartUiEventAddToCart,
+  CART_TOAST_STATE,
+  OrderEntry,
+} from '@spartacus/cart/base/root';
+import { AddedToCartToastComponentService } from './added-to-cart-toast-component.service';
+
+@Component({
+  selector: 'cx-added-to-cart-toast',
+  templateUrl: './added-to-cart-toast.component.html',
+})
+export class AddedToCartToastComponent implements OnDestroy {
+  protected subscription = new Subscription();
+
+  headerElement: HTMLElement | null;
+
+  toastContainerClass: string;
+
+  timeout: number;
+
+  toastContainerBaseClass = 'toast-container';
+
+  lastToastItem: CartToastItem;
+
+  scrollEventUnlistener: () => void;
+
+  @HostBinding('className') baseClass: string;
+
+  cartToasts$: Observable<CartToastItem[]> =
+    this.addedToCartToastService.getToasts();
+
+  constructor(
+    protected winRef: WindowRef,
+    protected renderer: Renderer2,
+    protected cd: ChangeDetectorRef,
+    protected addedToCartToastService: AddedToCartToastComponentService,
+    protected activeCartService: ActiveCartFacade,
+    protected el: ElementRef
+  ) {}
+
+  init(): void {
+    this.baseClass = 'cx-added-to-cart-toast';
```

Comment:
This baseClass thing doesn't look right.  Components are styled via a scss file for the component and are targetd via their selector anyways.  I don't understand the purpose of this implementation and it seems like the base styling should be done following the Spartacus convention.